package.json

```diff
@@ -24,15 +24,18 @@
     "url": "https://github.com/lennym/reqres/issues"
   },
   "homepage": "https://github.com/lennym/reqres",
-  "dependencies": {
-    "express": "^4.15.2",
-    "sinon": "^4.0.0"
+  "dependencies": {},
+  "peerDependencies": {
+    "express": "*",
+    "sinon": "*"
```

Comment:
I'm happy with using express as a peer dependency, but would prefer to leave sinon as a direct dependency. 

There's no particular reason to insist that an implementing project uses sinon as part of its unit test suite in order to use this library.